.github/workflows/merge-to-stage.js

```diff
@@ -150,6 +153,7 @@ const merge = async ({ prs, type }) => {
           merge_method: 'squash',
         });
       }
+      existingPRCount++
```

Comment:
Should this be brought into the `if (!process.env.LOCAL_RUN) {` conditional?

Reply:
No this should be fine - since you want to emulate the process "merging" PRs and stopping execution once the max is reached on a local run. Just without actually merging anything.